src/olympia/amo/tests/__init__.py

```diff
@@ -678,6 +682,14 @@ def addon_factory(
     for user in users:
         addon.addonuser_set.create(user=user)
 
+    application = version_kw.get('application', amo.FIREFOX.id)
+    if not category:
+        category = Category.from_static_category(
+            next(CATEGORIES[application][type_].itervalues()))
```

Comment:
They are, but `from_static_category()` only really does `return cls(**static_category.__dict__)` - It's used by some internal code to return fake `Category` instances that are never meant to be saved. 

Some tests use that as a shortcut, but you can easily do it yourself with something like (untested) `get_or_create(Category, id=data['id'], defaults=data)` where `data` is one of the items in `CATEGORIES[application][type_]`